internal/pkg/cli/app_delete.go

```diff
@@ -135,9 +138,10 @@ func newDeleteAppOpts(vars deleteAppVars) (*deleteAppOpts, error) {
 			}
 			return opts, nil
 		},
-		deletePipelineRunner: func() (cmd, error) {
+		pipelineDeleteExecutor: func(pipelineName string) (executor, error) {
```

Comment:
i love this interface name `executor` 
![skeletor](https://assets1.ignimgs.com/2013/08/27/skeletorjpg-e94010_160w.jpg?width=1280)